advisories/a63162.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Preliminary Technical Advisory 63162
+summary: Invalid incremental backups under certain circumstances
+toc: true
+---
+
+Preliminary date: April 6, 2021
+
+**Preliminary, and subject to change**
+
+## Description
+
+Cockroach Labs has discovered a bug relating to incremental backups, which under certain circumstances may result in data loss when restoring from an incremental backup. This is a **preliminary notice** in the interest of transparency, and will change.
+
+If an incremental backup coincides with an in-progress index creation or rebuild, `RESTORE`, or `IMPORT`, it is possible that the incremental backup will not include all of the indexed, restored or imported data.
```

Comment:
I don't think we have index "rebuilds"

Reply:
I was using ‘rebuild’ as synonymous with ‘backfill’, but I’ll move ‘rebuild’ since it’s not the right term.